src/cdk/overlay/position/flexible-connected-position-strategy.ts

```diff
@@ -366,6 +372,24 @@ export class FlexibleConnectedPositionStrategy implements PositionStrategy {
     return this;
   }
 
+  /**
+   * Sets the default offset for the overlay's connection point on the x-axis.
+   * @param offset New offset in the X axis.
+   */
+  withOffsetX(offset: number): this {
```

Comment:
Can we call these `withDefaultOffsetX`, `withDefaultOffsetY`? That will make it more clear that the offsets aren't additive.

Reply:
Sure, done.